ci_tutorial/packages_pipeline/multi_configuration.rst

```diff
@@ -0,0 +1,195 @@
+Package pipeline: multi configuration
+=====================================
+
+In the previous section we were building just 1 configuration. This section will cover the case in which we need to build more
+than 1 configuration. We will use the ``Release`` and ``Debug`` configurations here for convenience, as it is easier to 
+follow, but in real case these configurations will be more like Windows, Linux, OSX, building for different architectures, 
+cross building, etc.
+
+Let's begin cleaning our cache and initializing only the ``develop`` repo:
+
+
+.. code-block:: bash
+
+    $ conan remove "*" -c  # Make sure no packages from last run
+    $ conan remote remove "*"  # Make sure no other remotes defined
+    # Add develop repo, you might need to adjust this URL
+    $ conan remote add develop http://localhost:8081/artifactory/api/conan/develop
+
+
+We will create the packages for the 2 configurations sequentially in our computer, but note these will typically run
+in different computers, so it is typical for CI systems to launch the builds of different configurations in parallel.
+
+.. code-block:: bash
+    :caption: Release build
+
+    $ cd ai
+    $ conan create . --build="missing:ai/*" -s build_type=Release --format=json > graph.json
+    $ conan list --graph=graph.json --graph-binaries=build --format=json > built.json
```

Comment:
Not sure what does this mean in this context. We are not doing ``--build=missing``, but just for ``ai``. Dependencies aren't built and won't be uploaded.

Reply:
@memsharded 
Would it please be possible to also show how to decouple the `conan create` into `conan build` + `conan export-pkg` with the formatting? Should we create a graph for both of these commands then merge them?

It can be especially insteresting in CI, since creating an artifact with the whole conan cache (if built with `conan create`) can be cumbersome. This allows cases where one can create an artifact with only the `build` folder (if built with `conan build`)

Reply:
Hi @Todiq 

This would be a different topic than the one covered in this tutorial. This kind of things will probably follow in later versions, talking about implementation strategies like this.

However, in CI a ``conan build + conan export-pkg`` would be identical to ``conan create``, there will be no difference in computing time or storage in disk or anything like that. It only changes the location of the build folder, but actually it would be very recommended to use ``create`` because that garantees that future re-builds with ``conan install --requires=thispkg/version --build=thispkg/version`` work correctly, because those are also executed in the cache.

So I am not sure about the value of your proposal. ``conan build + conan export-pkg`` has some value for developers trying things locally, but it shouldn't have any advantage in the CI, I might be missing something.